### DISCUSSION TITLE
Cant enter list/array into custom parameter in Orion UI

### DISCUSSION THREAD

**Discussion by Wingmore** [PrefectHQ/prefect#8342] [Q&A]
I cant enter lists/array into the Orion UI Custom Parameters of deployments. Get an error if I try:

```
prefect.exceptions.ParameterTypeError: Flow run received invalid parameters:
 - capsule_ids: value is not a valid integer
 - capsule_ids: value is not a valid list
```

See also: https://discourse.prefect.io/t/flow-run-received-invalid-parameters-value-is-not-a-valid-list/2310

**Comment by serinamarie** [comment]
Hi @Wingmore, can you provide an [MRE](https://stackoverflow.com/help/minimal-reproducible-example) for this?

**Reply by Wingmore** [reply]
test_validation.py:
```
from typing import Union, List
from prefect import flow

@flow(log_prints=True)
def my_flow(
    some_list: Union[int, List[int]] = 5
):
    if type(some_list) == int:
        print(some_list)
    else:
        for i in some_list:
            print(i)

if __name__ == "__main__":
    my_flow([1,2,3])
```

Create Deployment
```
prefect deployment build test_validation.py:my_flow --name input_validation -a
```

In Orion UI I enter
![image](https://user-images.githubusercontent.com/36834195/218642281-9c1d586b-7eab-49a1-a0a9-8a08947cabe4.png)

When I hit run, I am greeted with this:
```
15:50:16.587 | ERROR   | Flow run 'psi4-ventani-b' - Validation of flow parameters failed with error:
Traceback (most recent call last):
  File "/home/usr/anaconda3/envs/mtt-prefect-env/lib/python3.9/site-packages/prefect/engine.py", line 296, in retrieve_flow_then_begin_flow_run
    parameters = flow.validate_parameters(flow_run.parameters)
  File "/home/usr/anaconda3/envs/mtt-prefect-env/lib/python3.9/site-packages/prefect/flows.py", line 339, in validate_parameters
    raise ParameterTypeError.from_validation_error(exc) from None
prefect.exceptions.ParameterTypeError: Flow run received invalid parameters:
 - some_list: value is not a valid integer
 - some_list: value is not a valid list
```

And this is what the parameter tab looks like.

![image](https://user-images.githubusercontent.com/36834195/218642365-e9ed75fc-32f8-431f-83bf-223e9447e128.png)

**Reply by Wingmore** [reply]
Note this may be an issue with `Union`... If I ONLY have a list, then it seems to works ok. That is, if I have `some_list: List[int]` as the arguments

**Reply by serinamarie** [reply]
I was able to reproduce the issue. I believe this discussion is relevant to the UI team, so I'll create an issue from this. For now, I'm glad you have a workaround!

![Screenshot 2023-02-14 at 11 44 29 AM](https://user-images.githubusercontent.com/42048900/218816618-623da139-7944-4e62-953c-f7b6b4a540d4.png)
